recipe/cmake-configure-alignment.patch

```diff
@@ -0,0 +1,63 @@
+--- CMakeLists.txt	2019-10-01 18:19:12.000000000 +0200
++++ CMakeLists.txt	2020-04-19 10:39:47.000000000 +0200
+@@ -98,6 +98,7 @@
+ SET(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake) # for FindReadline.cmake
+ 
+ SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -I${PROJECT_SOURCE_DIR}/src")
++SET(PACKAGE_VERSION 10.34)
+ 
+ # external packages
+ FIND_PACKAGE( BZip2 )
+@@ -540,16 +541,36 @@
+ # 8-bit library
+ 
+ IF(PCRE2_BUILD_PCRE2_8)
+-ADD_LIBRARY(pcre2-8 ${PCRE2_HEADERS} ${PCRE2_SOURCES} ${PROJECT_BINARY_DIR}/config.h)
+-SET_PROPERTY(TARGET pcre2-8
+-  PROPERTY COMPILE_DEFINITIONS PCRE2_CODE_UNIT_WIDTH=8)
++  ADD_LIBRARY(pcre2-8 ${PCRE2_HEADERS} ${PCRE2_SOURCES} ${PROJECT_BINARY_DIR}/config.h)
++
++SET_TARGET_PROPERTIES(pcre2-8 PROPERTIES
++  COMPILE_DEFINITIONS PCRE2_CODE_UNIT_WIDTH=8
++  MACHO_COMPATIBILITY_VERSION 10.0.0
++  MACHO_CURRENT_VERSION 10.0.0
++  VERSION 0.9.0
```

Comment:
Shouldn't this be `0.8.0`?

Reply:
No, they increased it to 9. See also the Debian list of files: https://packages.debian.org/de/sid/amd64/libpcre2-8-0/filelist Debian builds with autotools: https://salsa.debian.org/debian/pcre2/-/blob/master/debian/rules#L26 I'm a bit confused that ABI Laboratory reports differently.

Reply:
Okay. I see now I was looking at the wrong source.